docs/dev_guide/diagnostic_ref/dpct1013.rst

```diff
@@ -25,3 +25,43 @@ Suggestions to Fix
 ------------------
 
 Review the logic and adjust it.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   __global__ void foo(double *res) {
+     double a = 1.00001;
+     double b = 100000000000000000;
+     *res = __dadd_ru(a, b);
+   }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo(double *res) {
+     double a = 1.00001;
+     double b = 100000000000000000;
+     /*
+     DPCT1013:0: The rounding mode could not be specified and the generated code
+     may have different accuracy than the original code. Verify the correctness.
+     SYCL math built-in function rounding mode is aligned with OpenCL C 1.2
+     standard.
+     */
+     *res = a + b;
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo(double *res) {
+     double a = 1.00001;
+     double b = 100000000000000000;
+     *res = sycl::nextafter(a + b, (double)INFINITY);

```

Comment:
@zhiweij1 , please add explain to L65 (why not migrate to it directly)

Reply:
1. This migration was implemented a long time ago. Maybe we did not find this way to migrate at that time. This PR is focus on the manually fix example, so I did not update the migration.
2. Maybe user does not care about the accuracy. Then migrating to `a+b` is enough.

Reply:
Added.